css/product-payment.css

```diff
@@ -0,0 +1,555 @@
+  width: 22px;
+  border: 1px solid var(--color-solid);
+  margin-left: 5px;
+}
+
+.product-page__quantity {
+  font-size: var(--twenty-pixel);
+  font-weight: var(--font-weight);
+  line-height: 25px;
+  text-align: center;
+  width: 95px;
+  padding: 7px 10px 10px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-right: 5px;
+  border-left: 1.5px solid var(--color-solid);
+  border-right: 1.5px solid var(--color-solid);
+  color: var(--color-solid);
+  background-color: var(--color-whiteX);
+}
+
+.product-page__increase{
+  border: none;
+  background: none;
+  cursor: pointer;
+  width: 25px;
+  border: 1.5px solid var(--color-solid);
+  margin-right: 6px;
+}
+
+.product-page__increase--1{
+  height: var(--eighteen-pixel);  
+  width: var(--zero);
+  position: absolute;
+  margin-left: 93px;
+}
+
+.product-page__add-purchases{
+  display: flex;
+  gap: var(--eight-pixel);
+  background-color: var(--color-pink);
+  border: none;
+  border-radius: var(--eight-pixel);
+  cursor: pointer;
+  box-shadow: 0px 4px 12px 0px #D14D411A;
+  padding: 10px 20px;
+  position: relative;
+  margin-top: -5px;
+  margin-right: 20px;
+}
+
+.product-page__add-purchases:hover{
+  background-color: #9e255b;
+}
+
+.product-page__add-cart{
+  font-size: var(--font-size-smalls);
+  font-weight: var(--font-weight-small);
+  line-height: var(--eighteen-pixel);
+  text-align: center;
+  color: var(--color-white);
+  letter-spacing: 0.06em;
+}
+
+.product-page__payment-methods {
+  display: flex;
+  height: 9px;
+  gap: 5px;
+  margin-top: 10px;
+}
+
+.product-page__add-target{
+  margin-left: 150px;
+}
+
+/* Desktop style */
+@media (min-width: 768px) {
+  .product-page__top {
+    padding-bottom: 24px;
+    margin: 0 5px; 
+  }
+
+  .product-page__text {
+    font-size: var(--font-size-smalls);
+  }
+  
+  .product-page__content{
+    flex-direction: row;
+    margin: 0px 0px 20px;
+    gap: 52px;
+  }
+
+  .product-page__air {
+    margin-top: 21pc;
+  }
+  
+  .product-page__image {
+    width: 150px;
+  }
+
+  .product-page__zoom {
+    width: 640px;
+  }
+
+  .product-page__img {
+    align-items: center;
+    margin: auto;
+    width: 430px;
+  }
+
+  .product-page__card{
+    width: 437px;
+    padding: 10px 18px 0px;
+  }
+
+  .product-page__full{
+    padding: 0px;
+}
+
+  .product-page__description{
+    font-size: 28px;
+  }
+
+  .product-page__stars{
+    font-size: 23px;
+  }
+
+  .product-page__count{
+    font-size: var(--twenty-pixel);
+  }
+
+  .product-page__write{
+    font-size: var(--font-size-smalls);
+  }
+
+  .product-page__badge{
+    margin-left: 13px;
+    padding: 5px 13px;
+  }
+
+  .product-page__extend{
+    gap: var(--eighteen-pixel);
+  }
+
+  .product-page__warranty{
+    font-size: var(--sixteen-pixeles);
+  }
+
+  .product-page__requires{
+    font-size: var(--font-size-small);
+  }
+
+  .product-page__bords{
+    width: 420px;
+  }
+
+  .product-page__add{
+    margin-left: -18px;
+    width: 437px;
```

Comment:
No lo necesitas, con margin right negativo sirve

<img width="909" alt="image" src="https://github.com/user-attachments/assets/d6f4518f-3e63-42f7-9348-683ae1e13e38">

```suggestion
    margin-right: -18px;
```